requirements.txt

```diff
@@ -3,18 +3,16 @@ backports.unittest-mock
 click
 docker[ssh]>=4.4.4
 flake8
-pyOpenSSL>=17.0.0
+pyOpenSSL>=20.0.1
 python-dotenv
 requests>=2.25.1
-six
 applicationinsights
 prompt_toolkit
 rope
 tox
-pyyaml==5.4
+pyyaml==5.3.1
```

Comment:
why are we going down to a lower version of pyyaml here?

Reply:
that's the highest we can go to support py35